install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,431 @@
+      size: 15GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+    - name: openshift
+      size: 30GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+  state: running
+  cloud_init:
+    root_password: "{{ root_password }}"
+    authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+    custom_script: "{{ cloud_init_script_node | to_nice_yaml }}"
+
+openshift_ovirt_vm_profile:
+  master_vm:
+    <<: *node_item
+    memory: 16GiB
+    cores: "{{ vm_cores | default(4) }}"
+    disks:
+      - name: docker
+        size: 15GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: openshift_local
+        size: 30GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: etcd
+        size: 25GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+    cloud_init:
+      root_password: "{{ root_password }}"
+      authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+      custom_script: "{{ cloud_init_script_master | to_nice_yaml }}"
+  node_vm:
+    <<: *node_item
+  etcd_vm:
+    <<: *node_item
+  lb_vm:
+    <<: *node_item
+
+cloud_init_script_node: &cloud_init_script_node
+  packages:
+    - ovirt-guest-agent
+  runcmd:
+    - sed -i 's/# ignored_nics =.*/ignored_nics = docker0 tun0 /' /etc/ovirt-guest-agent.conf
+    - systemctl enable ovirt-guest-agent
+    - systemctl start ovirt-guest-agent
+    - mkdir -p /var/lib/docker
+    - mkdir -p /var/lib/origin/openshift.local.volumes
+    - /usr/sbin/mkfs.xfs -L dockerlv /dev/vdb
+    - /usr/sbin/mkfs.xfs -L ocplv /dev/vdc
+  mounts:
+    - [ '/dev/vdb', '/var/lib/docker', 'xfs', 'defaults,gquota' ]
+    - [ '/dev/vdc', '/var/lib/origin/openshift.local.volumes', 'xfs', 'defaults,gquota' ]
+  power_state:
+    mode: reboot
+    message: cloud init finished - boot and install openshift
+    condition: True
+cloud_init_script_master:
+  <<: *cloud_init_script_node
+  runcmd:
+    - sed -i 's/# ignored_nics =.*/ignored_nics = docker0 tun0 /' /etc/ovirt-guest-agent.conf
+    - systemctl enable ovirt-guest-agent
+    - systemctl start ovirt-guest-agent
+    - mkdir -p /var/lib/docker
+    - mkdir -p /var/lib/origin/openshift.local.volumes
+    - mkdir -p /var/lib/etcd
+    - /usr/sbin/mkfs.xfs -L dockerlv /dev/vdb
+    - /usr/sbin/mkfs.xfs -L ocplv /dev/vdc
+    - /usr/sbin/mkfs.xfs -L etcdlv /dev/vdd
+  mounts:
+    - [ '/dev/vdb', '/var/lib/docker', 'xfs', 'defaults,gquota' ]
+    - [ '/dev/vdc', '/var/lib/origin/openshift.local.volumes', 'xfs', 'defaults,gquota' ]
+    - [ '/dev/vdd', '/var/lib/etcd', 'xfs', 'defaults,gquota' ]
+----
+
+<1> `<qcow_url>` is the download URL of the *_RHEL-guest-image_*. The *_RHEL-guest-image_* includes the `cloud-init` package, which is required by this playbook. (If you are using a different operating system, you must install the `cloud-init` package separately.)
```

Comment:
You need it twice: (1) to install the bastion VM and (2) to install RHV. You can't use the same URL because it will have expired. It's only to make sure that the RHEL guest image is installed. Does not need to be the same URL but DOES need to be valid RHEL guest image URL.

Reply:
Then you need to point out that you must obtain this URL again and cannot use the one that you used earlier. It's not clear that you need to use this URL within minutes of generating it - from reading it, I assumed that you'd be able to reuse the URL. I fear that users would do the same.